showkase-annotation/src/main/java/com/airbnb/android/showkase/annotation/ShowkaseCodegenMetadata.kt

```diff
@@ -34,5 +34,6 @@ annotation class ShowkaseCodegenMetadata(
     val showkaseWidthDp: Int = -1,
     val showkaseHeightDp: Int = -1,
     val insideWrapperClass: Boolean = false,
-    val insideObject: Boolean = false
+    val insideObject: Boolean = false,
+    val previewParameterClass: Array<KClass<*>> = [],
```

Comment:
can't this be a an array of `PreviewParameter`? it seems like it should be able to be more specific (assuming the annotation processor has the compose library available at runtime as a dependency

Reply:
the annotation processor does not have the compose library available at runtime. This was mainly due to how it requires a custom compiler and hence it wasn't possible to use compose without adding android dependencies. This might change in the future though or might've already change now @elihart